src/PokeDetail.js

```diff
@@ -22,7 +23,9 @@ class PokeDetail extends Component {
             <section>
                 <h1>{data.pokemon}</h1>
                 {/* <p>Poke ID: {id}</p> */}
-                <img src={data.url_image} alt='poke' />
+                <div className='poke-detail'>
+                    <img src={data.url_image} alt='poke' />
```

Comment:
Here you'll want to pull some additional information about the pokemon beyond its name and img(for example, their hp or special abilities). But you can pull the other info in just like you did for those two.